aries-site/src/pages/templates/forms.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { Meta, SubsectionText } from '../../components';
+import { SignInExample, SignUpExample } from '../../examples';
+import { ContentSection, Example, Layout, Subsection } from '../../layouts';
+import { getPageDetails } from '../../utils';
+
+const title = 'Forms';
+const topic = 'Templates';
+const page = getPageDetails(title);
+
+const Forms = () => {
+  return (
+    <Layout title={title}>
+      <Meta
+        title={title}
+        description={page.seoDescription}
+        canonicalUrl="https://design-system.hpe.design/templates/forms"
+      />
+      <ContentSection>
+        <Subsection name={title} level={1} topic={topic}>
+          <SubsectionText>
+            Template patterns for common form use cases, from application
+            configurations to payment acceptance for the latest as-a-service
+            product offerings.
+          </SubsectionText>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Sign Up">
+          <Example
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/templates/forms/SignUpExample.js"
+            figma="https://www.figma.com/file/K0PppsSh6aQSszqlwKAekg/hpe-design-system-stickers?node-id=259%3A86"
```

Comment:
Figma files are not opening for me so not sure if this is an overall problem we can discuss with @gregfuruiye

Reply:
Oooh. Good to know. @L0ZZI , thoughts on how we handle? Option A: create a shareable view only version of this file? Option B: Remove Figma link for now? Others?